files/scripts/profiling/deployment_time_profiling.py

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+import pandas as pd
+from datetime import datetime
+import sys
+
+def help():
+    print("\nThe script requires exactly 2 arguments in sequence\n1.Input file\n2.Output file")
```

Comment:
Use f-string and a multiline string. Avoid multiple print statements.
If possible use a console logger instead of a print statement.